core/templates/pages/exploration-player-page/layout-directives/audio-bar.directive.html

```diff
@@ -252,6 +252,11 @@
     top: 3.5em;
   }
 
+  @media(max-width: 768px) {
+    .headroom--unpinned.audio-header.audio-header-margin {
```

Comment:
Start with a class we have defined. e.g. `.audio-header ...`

Reply:
done.